qutebrowser/misc/earlyinit.py

```diff
@@ -172,7 +172,7 @@ def check_qt_version():
     """Check if the Qt version is recent enough."""
     from PyQt5.QtCore import (qVersion, QT_VERSION, PYQT_VERSION,
                               PYQT_VERSION_STR)
-    from pkg_resources import parse_version
+    from qutebrowser.utils.qtutils import parse_version
```

Comment:
This isn't going to work out - since this code is checking whether the Qt version is recent enough, we can't import things from `qtutils` here (and we can't even be sure `QVersionNumber` exists, though it was added in Qt 5.6 which is rather old).

I guess a proper solution would be to use `QVersionNumber` directly, but in a way that we still show a proper error if it isn't available. However, I'm already changing the code below in my `drop-qt` branch - so I think to avoid conflicts it'd be best to keep `pkg_resources` here for now, but with a `# FIXME: Replace by QVersionNumber` comment or so.

Reply:
Thank you for the speedy feedback! So even "Qt < 5.12" will be dropped. When is that going to happen? I wonder if I should move forward to improve #5495. Maybe when that work is done, drop-qt branch has already been merge. Then "# FIXME: Replace by QVersionNumber" isn't needed.

Reply:
Probably somewhen over the next couple of days/weeks - some things broke with it, and I want to take care of remaining issues for v1.14.1 (and some non-qutebrowser deadlines) before continuing with it.

I'd rather like to get things in one by one instead of tangling things together which can be separate - as long as we're still using `pkg_resources` for resources anyways, the FIXME doesn't hurt anyone.

Reply:
FWIW I did push the Qt drop to `master` now. After rebasing this PR, I guess you could do something like this here, to avoid depending on things which might not be available (untested):

```python3
try:
    from PyQt5.QtCore import QVersionNumber, QLibraryInfo
    recent_qt = QLibraryInfo.version().normalized() >= QVersionNumber(5, 12)
except (ImportError, AttributeError):
    # QVersionNumber was added in Qt 5.6, QLibraryInfo.version() in 5.8
    recent_qt = False

if QT_VERSION < 0x050C00 or PYQT_VERSION < 0x050C00 or not recent_qt:
    # ...
```